commands.js

```diff
@@ -1465,6 +1465,26 @@ var commands = exports.commands = {
 	},
 	forcerenamehelp: ["/forcerename OR /fr [username], [reason] - Forcibly change a user's name and shows them the [reason]. Requires: % @ & ~"],
 
+	hidetext: function (target, room, user) {
+		if (!target) return this.parse('/help hidetext');
+
+		var reason = this.splitTarget(target);
+		var targetUser = this.targetUser;
+		var name = this.targetUsername;
+		if (!targetUser) return this.errorReply("User '" + name + "' does not exist.");
+		if (!this.can('lock', targetUser) && !this.can('ban', targetUser, room)) return false;
```

Comment:
Don't use `this.can` with `&&`. You'll need to use `user.can` and write your own error message.